src/components/ServiceBanner.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, Typography } from '@mui/material';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { MaintenanceMessageContent } from '../pages/errorpages/MaintenanceMessagePage';
+import { LocalStorageKey } from '../utils/constants';
+import { getMaintenanceInfo } from '../utils/status-message-helpers';
+
+const prodHostname = 'nva.sikt.no';
+const hostname = window.location.hostname;
+const isTestEnvironment = hostname !== prodHostname;
+const defaultBannerState = localStorage.getItem(LocalStorageKey.EnvironmentBanner);
+
+enum ServiceBannerVisibility {
+  Minimized = 'minimized',
+  Normal = 'normal',
+}
+
+export const ServiceBanner = () => {
```

Comment:
Denne får nå en cc=11, men jeg syns alle utbedringer for det gjør mer skade enn nytte..
Kan feks endre til dette:

```
// Utenfor komponenten:
const shouldShowMaintenanceInfo = (maintenanceInfo: MaintenanceInfo | null) => {
  return maintenanceInfo && maintenanceInfo.severity !== 'block';
};

// I komponenten:
const showMaintenanceInfo = shouldShowMaintenanceInfo(maintenanceInfo);
```

Men da syns jeg at dette er mye bedre, selv med høyere cc
```
const showMaintenanceInfo = maintenanceInfo && maintenanceInfo.severity !== 'block';
```

Reply:
Tror jeg begynner å lene mer mot å øke threshold for cc, til feks 15 🤔

Reply:
Dette er en ny fil og relativt liten, men selv her syns jeg å håndheve maks 10 bare ville gjort det mindre lesbart. Så tenker egentlig å endre 15-20 er bedre